libs/shared/ui/src/lib/components/navigation-left/navigation-left.tsx

```diff
@@ -38,6 +38,7 @@ export type NavigationLeftLinkProps = {
       }
     | {
         iconName?: IconName
+        iconStyle?: 'solid' | 'regular' | 'brands'
```

Comment:
You can use the `IconStyle` from the lib `@fortawesome/fontawesome-common-types`